packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -619,7 +619,7 @@ export default class Meeting extends StatelessWebexPlugin {
      * @public
      * @memberof Meeting
      */
-    this.recording = null;
+    this.recording = attrs.locus?.controls.record ? {state: RECORDING_STATE.IDLE, modifiedBy: '', lastModified: ''} : null;
```

Comment:
I do prefer to define classes for different controls with an abstract called `Control` and have classes `Recording` `Locked` etc etc.